### PR TITLE
Update installer images and tweak machine names

### PR DESCRIPTION
**Pull request by codyhoag**
Previews:
- New [install-config image](https://deploy-preview-36506--osdocs.netlify.app/openshift-enterprise/latest/architecture/architecture-installation.html#installation-overview_architecture-installation) (original [here](https://docs.openshift.com/container-platform/4.8/architecture/architecture-installation.html#installation-overview_architecture-installation))
- New [Machine creation image](https://deploy-preview-36506--osdocs.netlify.app/openshift-enterprise/latest/architecture/architecture-installation.html#installation-process_architecture-installation) (original [here](https://docs.openshift.com/container-platform/4.8/architecture/architecture-installation.html#installation-process_architecture-installation))

### CODE REVIEW

**Comment by codyhoag** [comment]
@gpei can you verify? Thanks!

**Comment by gpei** [comment]
LGTM

File: modules/installation-process.adoc [hunk 1]
```diff
@@ -63,15 +63,15 @@ If you do not use infrastructure that the installation program provisioned, you
 * Cluster networking, including the DNS records and required subnets
 * Storage for the cluster infrastructure and applications
 
-If your cluster uses user-provisioned infrastructure, you have the option of adding RHEL worker machines to your cluster.
+If your cluster uses user-provisioned infrastructure, you have the option of adding {op-system-base} compute machines to your cluster.
 
 [discrete]
 == Installation process details
 
 Because each machine in the cluster requires information about the cluster when it is provisioned, {product-title} uses a temporary _bootstrap_ machine during initial configuration to provide the required information to the permanent control plane. It boots by using an Ignition config file that describes how to create the cluster. The bootstrap machine creates the control plane machines that make up the control plane. The control plane machines then create the compute machines, which are also known as worker machines. The following figure illustrates this process:
```
Comment (bmcelvee) [inline modules/installation-process.adoc:71]: Spotted a sneaky "worker machines" under _Installation process details_ in the preview.

**Comment by bmcelvee** [comment]
I thought I spotted a sneaky "worker machines", but it was meant to be there. :sweat_smile: LGTM!

**Pull request merged**